### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by jschueller**
- Fix warnings from gcc13
- Add CalibrationResult::drawResidualsNormalPlot @YomGarcia 
residualnormalplot

- Fix GridLayout serialization @adumasphi 
- Fix loading of old kriging studies @sofianehaddad 
- Add SpecFunc::Clip01
- Tweak the 1-d minimum volume computation optimization starting point depending on the skewness, now triangular & truncated testcases give a correct (non-empty) interval
- Fix some methods visibility, eg make UserDefined::computeRange private

### CODE REVIEW

**Review by mbaudin47 — CHANGES_REQUESTED** [review]

**Review by sofianehaddad — APPROVED** [review]

**Comment by sofianehaddad** [comment]
Nice catch in `Kriging` thanks

File: python/src/SpecFunc_doc.i.in [hunk 1]
```diff
@@ -1200,3 +1200,24 @@ Returns
...
+
+%feature("docstring") OT::SpecFunc::Clip01
+"Clip a value on [0, 1].
+
+Parameters
+----------
+p : float
+    Value to clip
+tail : bool, default=False
+    Whether to return 1-p or not.
```
Comment (mbaudin47) [inline python/src/SpecFunc_doc.i.in:1213]:
```suggestion
    If False, then return min(1, max(0,p)). Otherwise return 1 - min(1, max(0,p)). 
```

**Comment by jschueller** [comment]
I put the image in the description
the circleci redirector stuff is broken atm

**Review by regislebrun — APPROVED** [review]

**Comment by mbaudin47** [comment]
I asked for a change:

image

and was clear about it:

image

so i guess that you forgot about it. Can you please revert that merge?

**Comment by jschueller** [comment]
too late to revert, I guess we can do that in another branch

**Pull request merged**